src/app/test-record/defects/defects.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, OnInit, ChangeDetectionStrategy, Input} from '@angular/core';
+import {TestType} from '@app/models/test.type';
+import {TestResultModel} from '@app/models/test-result.model';
+
+@Component({
+  selector: 'vtm-defects',
+  templateUrl: './defects.component.html',
+  styleUrls: ['./defects.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DefectsComponent implements OnInit {
```

Comment:
you should be able to write test to mock these @Input(). Please check the ADR branch or lets talk